Source/Basic Shapes/SvgLine.Drawing.cs

```diff
@@ -0,0 +1,42 @@
+#if !NO_SDC
+using System.Drawing;
+using System.Drawing.Drawing2D;
+
+namespace Svg
+{
+    public partial class SvgLine : SvgMarkerElement
+    {
+        private GraphicsPath _path;
+
+        public override System.Drawing.Drawing2D.GraphicsPath Path(ISvgRenderer renderer)
```

Comment:
If it can be omitted, please change it in this PR.

`System.Drawing.Drawing2D.GraphicsPath` -> `GraphicsPath`

Reply:
Done